test_unstructured/metrics/test_evaluate.py

```diff
@@ -17,6 +18,35 @@
 
 UNSTRUCTURED_OUTPUT_DIRNAME = "unstructured_output"
 GOLD_CCT_DIRNAME = "gold_standard_cct"
+GOLD_ELEMENT_TYPE_DIRNAME = "gold_standard_element_type"
+
+
+@pytest.mark.skipif(is_in_docker, reason="Skipping this test in Docker container")
+def test_text_extraction():
+    output_dir = os.path.join(TESTING_FILE_DIR, UNSTRUCTURED_OUTPUT_DIRNAME)
+    source_dir = os.path.join(TESTING_FILE_DIR, GOLD_CCT_DIRNAME)
+    export_dir = os.path.join(TESTING_FILE_DIR, "test_evaluate_results_cct")
+    measure_text_extraction_accuracy(
+        output_dir=output_dir, source_dir=source_dir, export_dir=export_dir
+    )
+    df = pd.read_csv(os.path.join(export_dir, "all-docs-cct.tsv"), sep="\t")
+    assert len(df) == 3
+    assert len(df.columns) == 5
```

Comment:
will this columns value always be 5? I know we have a lot of changes slated for this output file. What exactly is this test testing, and can we update the test name to reflect that?

Reply:
As long as we don't make any changes to `measure_text_extraction_accuracy` function, the column will always be 5 ("filename", "doctype", "connector", "cct-accuracy", "cct-%missing"). This is just testing the `measure_text_extraction_accuracy` in general if it behave and save as expected. I can add another assertion to check if the file existed before all the file opening if that makes more sense.